tools/walletextension/main/cli.go

```diff
@@ -59,6 +59,18 @@ const (
 	storeIncomingTxs        = "storeIncomingTxs"
 	storeIncomingTxsDefault = true
 	storeIncomingTxsUsage   = "Flag to enable storing incoming transactions in the database for debugging purposes. Default: true"
+
+	rateLimitThresholdName    = "rateLimitThreshold"
+	rateLimitThresholdDefault = 1000
```

Comment:
If this is 200 and the decay is 1, we allocate 200ms/per second to each user.

Reply:
If the decay is 1 then every ms one ms is subtracted from the score.
Setting `decay` to `0.2` means that every second it decays for 200ms. 
Setting `threshold` to 1000 means that users can use 1000ms in a second, but it will decay with the decay rate above. So we are not rate limiting the user if he created a request that takes `250ms` and wants to call another request immediately after that. But if he/she keeps doing it rate limiting will kick in.